index.js

```diff
@@ -1,11 +1,11 @@
 /* global window, define */
-(function(factory) {
+(function (factory) {
```

Comment:
@chetanism thanks for the PR! You've introduced a lot of noise into this PR by changing the coding style. There's things I don't like about this coding style, but I wouldn't go ahead and change it in a PR without speaking to the author first.